cmd/minikube/cmd/root_test.go

```diff
@@ -44,3 +46,29 @@ func TestPreRunDirectories(t *testing.T) {
 		}
 	}
 }
+
+func cleanup() {
+	pflag.Set("v", "0")
+	pflag.Lookup("v").Changed = false
+}
+
+func TestFlagShouldOverrideConfig(t *testing.T) {
```

Comment:
Good catch - that case wasn't actually working.  I've updated and rebased the commit (maybe I shouldn't have?)